<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,5">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Vertical node count</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Horizontal node count</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Initial temperature</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Element K factor</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Heat capacity</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Density</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Heat transfer coefficient (alpha)</string>
        </property>
       </widget>
      </item>
      <item row="18" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Ambient temperature</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QDoubleSpinBox" name="heightInput">
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QDoubleSpinBox" name="widthInput">
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QSpinBox" name="verticalNodeCountInput">
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QSpinBox" name="horizontalNodeCountInput">
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QDoubleSpinBox" name="initialTemperaturInput">
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>30.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QDoubleSpinBox" name="kFactorInput">
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QDoubleSpinBox" name="heapCapacityInput">
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>700.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="15" column="0" colspan="2">
       <widget class="QDoubleSpinBox" name="densityInput">
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>7800.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="17" column="0" colspan="2">
       <widget class="QDoubleSpinBox" name="alphaInput">
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>25.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="19" column="0" colspan="2">
       <widget class="QDoubleSpinBox" name="ambientTemperatureInput">
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>1200.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="20" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item row="22" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Time step</string>
        </property>
       </widget>
      </item>
      <item row="21" column="0" colspan="2">
       <widget class="QDoubleSpinBox" name="timeInput">
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>500.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="23" column="0" colspan="2">
       <widget class="QDoubleSpinBox" name="timeStepInput">
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>50.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item row="3" column="6">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="rightBC">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="leftBC">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="topBC">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Boundary conditions</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="Line" name="line_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="bottomBC">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="5">
       <widget class="Line" name="line_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="simulateButton">
        <property name="text">
         <string>Simulate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="QTableView" name="resultsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>widthInput</tabstop>
  <tabstop>heightInput</tabstop>
  <tabstop>verticalNodeCountInput</tabstop>
  <tabstop>horizontalNodeCountInput</tabstop>
  <tabstop>initialTemperaturInput</tabstop>
  <tabstop>kFactorInput</tabstop>
  <tabstop>heapCapacityInput</tabstop>
  <tabstop>densityInput</tabstop>
  <tabstop>alphaInput</tabstop>
  <tabstop>ambientTemperatureInput</tabstop>
  <tabstop>timeInput</tabstop>
  <tabstop>timeStepInput</tabstop>
  <tabstop>topBC</tabstop>
  <tabstop>rightBC</tabstop>
  <tabstop>bottomBC</tabstop>
  <tabstop>leftBC</tabstop>
  <tabstop>simulateButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
